dotCMS/src/main/java/com/dotmarketing/quartz/job/CleanUpFieldReferencesJob.java

```diff
@@ -101,34 +113,26 @@ public void run(JobExecutionContext jobContext) throws JobExecutionException {
             Logger.error(CleanUpFieldReferencesJob.class,
                     "Error cleaning up field references. Field velocity var: " + field.variable(), e);
         }
+*/
+
+        try {
+            Thread.sleep(25000);
```

Comment:
@fabrizzio-dotCMS did you leave your test sleep in here?

Reply:
it's a draft I forgot to set that back to normal. yeah, that was to simulate a long-running task. I still need to visit the other jobs. I'll be doing internal QA for a short while before I do that.